packages/e2e-test-utils/src/canvas.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Gets the editor canvas frame.
+ */
+export function canvas() {
```

Comment:
maybe getEditorCanvas or something more descriptive

Reply:
I just wanted to keep it small like `page` so it doesn't trigger reformatting in the many places we need to update e2e tests for the post editor later. See #21102.